### ISSUE TITLE
Can't DELETE in Neo Environment

### ISSUE THREAD

**Issue by DevGregor** [dbeck121/CPI-Helper-Chrome-Extension#164]
I recently added a new Plugin "Unlock Plugin" (#163 ) and for some reason I the makeCallPormise function doesn't work in the Neo environment when the HTTP method "DELETE" is used.

I'm currently trying to figure out what the cause is but haven't found anything yet. In CF it works just fine.

**Comment by dbeck121** [comment]
I will check too but I do not see a reason except for differences in API. Maybe you used a ring url? Some APIs use /itspaces in the name. 

What is the error message?

**Comment by DevGregor** [comment]
> I will check too but I do not see a reason except for differences in API. Maybe you used a ring url? Some APIs use /itspaces in the name.
> 
> What is the error message?

I tracked the used URL with the Network tab while inspecting and it is the same request. Note that I am also able to request all locked artifacts so this should work.

The only difference I was able to make out was the following:
CF -> /api/v1/IntegrationDesigntimeLocks('${resourceId}')
Neo -> /api/v1/IntegrationDesigntimeLocks(ResourceId='${resourceId}')

The Neo version works for both but the response was always the same in the Neo environment, namely:
403 Forbidden

Am I using the makePromiseCall function the wrong way here?
